go/test/endtoend/vtgate/schematracker/restarttablet/schema_restart_test.go

```diff
@@ -91,10 +90,7 @@ func TestMain(m *testing.M) {
 
 		// restart the tablet so that the schema.Engine gets a chance to start with existing schema
 		tablet := clusterInstance.Keyspaces[0].Shards[0].PrimaryTablet()
-		tablet.VttabletProcess.ExtraArgs = []string{
-			"--queryserver-config-schema-change-signal",
-			fmt.Sprintf("--queryserver-config-schema-change-signal-interval=%d", signalInterval),
-		}
+		tablet.VttabletProcess.ExtraArgs = append(tablet.VttabletProcess.ExtraArgs, "--queryserver-config-schema-change-signal")
```

Comment:
We are no longer using the signal interval is that expected? If yes, we should remove the definition of `signalInterval` on line `45`.